toDoApp/src/screens/home/List/ListItem/Check/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Pressable, Image, View } from 'react-native'
+
+export const Check: React.FC<{
+  value: boolean
+  onPress: (id: number) => () => void
+  id: number
+}> = ({ value, onPress, id }) => {
+  return (
+    <View>
+      <Pressable onPress={onPress(id)}>
+        <Image
+          source={
+            value
+              ? require('../../../../../assets/images/iconCheckboxActive.png')
+              : require('../../../../../assets/images/iconCheckboxInactive.png')
```

Comment:
Two things here.
1 - Maybe you could extract this to a value above the render.
2 - `require` its not a ES6 thing, you should use `import`

Reply:
I fixed 1), but for some reason i'm unable to import the image from that same route. I guess I'm doing something wrong but I can't seem to find what it is.